pkgs/development/tools/misc/sysbench/default.nix

```diff
@@ -16,6 +16,8 @@ stdenv.mkDerivation rec {
     sha256 = "1sanvl2a52ff4shj62nw395zzgdgywplqvwip74ky8q7s6qjf5qy";
   };
 
+  configureFlags = [ "--with-gcc-arch=unknown" ];
```

Comment:
Not sure if this should depend on ``, enableAVX ? stdenv.hostPlatform.avxSupport``.

Reply:
I'm not sure it should depend on the host platform... I guess it could depend one the target platform, but then, what's the default value for `avxSupport` for the hydra builds?

Reply:
With AVX I think https://github.com/NixOS/nixpkgs/blob/f0efbe21f9ae1e17d1aca922708f1bdef605039d/lib/systems/architectures.nix#L100